evergreen.yml

```
@@ -657,6 +662,10 @@ functions:
         working_dir: mongosql-odbc-driver
         script: |
           ${prepare_shell}
+
+          # change mongosql version in core/Cargo.toml from branch to tag
+          sed -i "s/branch = \"main\"/tag = \"${MONGOSQL_VERSION}\"/" core/Cargo.toml
```

```suggestion
          sed -i "/^mongosql = {.*branch = \"main\".*}/ s/branch = \"main\"/tag = \"${MONGOSQL_VERSION}\"/" core/Cargo.toml
```
Added a check for `mongosql` in the line.

---

Updated. The syntax differs slightly but I verified in the logs it compiles the right version